cmd/maya-exporter/app/collector/zvol/pool_synctime_test.go

```diff
@@ -104,6 +112,8 @@ func TestPoolSyncTimeMetricCollector(t *testing.T) {
 			regex := mockServer.BuildRegex(out)
 			vol := NewPoolSyncMetric(tt.runner)
 			stop := make(chan struct{})
+			os.Setenv("HOSTNAME", tt.hostname)
+			defer os.Unsetenv("HOSTNAME")
```

Comment:
probably better to set it back to whatever was the value prior to running this test.

Reply:
ok.